### PR TITLE
Issue #159 Document windows + virtualbox driver issue

### PR DESCRIPTION
**Pull request by praveenkumar**
Document around virtualbox issue with windows.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,8 +48,10 @@ The driver can be selected via the `--vm-driver=xxx` flag of `minishift start`.
 [docker machine drivers](./docs/docker-machine-drivers.md) for more details on supported drivers
 and how to install them.
 
-**Note**: For most hyervisor VT-x/AMD-v virtualization must be enabled in BIOS. For Hyper-V, however,
+**Note**:
+- For most hyervisor VT-x/AMD-v virtualization must be enabled in BIOS. For Hyper-V, however,
 it needs to be disabled.
+- We recommend to use `Virtualbox 5.1.12` for Windows to avoid [known issue](./docs/troubleshooting.md/#troubleshooting.md#error-getting-state-for-host-machine-does-not-exist)
```
Comment (hferentschik) [inline README.md]: Should we say "at least" or "Virtualbox > 5.1.12"?

Comment (hferentschik) [inline README.md]: "on Windows"!?

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -73,3 +75,14 @@ entry section from `dhcpd_leases` file. Finally, reboot your system.
...
 **Note:** You can completely reset IP database by removing both the files
 manually but this is very **risky**.
+
+<a name="virtualbox-driver"></a>
+## virtualbox driver
+
+<a name="error-getting-state-for-host-machine-does-not-exist"></a>
+### Error: getting state for host: machine does not exist
+
+The problem is likely to be either you have a outdated virtualbox installed or
```
Comment (hferentschik) [inline docs/troubleshooting.md]: If you are using Windows, the problem is likely to be either  an outdated version of Virtual Box or you forgot to use `--vm-driver virtualbox` option when starting _minishift_.

File: README.md (hunk 1 above)
Comment (budhrg) [inline README.md]: We might be confusing this version issue with https://github.com/minishift/minishift/issues/236.

Even, @agajdosi reported same here https://github.com/minishift/minishift/issues/159#issuecomment-269957348.

For me I couldn't reproduce issue even with `5.0.30` if running from `%USERPROFILE%` containiing drive.

This forcing me to think whether we require this note or not. 

However, I would suggest to have proper testing around `%USERPROFILE%` and check.

File: README.md [hunk 3]
```diff
@@ -48,8 +48,10 @@ The driver can be selected via the `--vm-driver=xxx` flag of `minishift start`.
 [docker machine drivers](./docs/docker-machine-drivers.md) for more details on supported drivers
 and how to install them.
 
-**Note**: For most hyervisor VT-x/AMD-v virtualization must be enabled in BIOS. For Hyper-V, however,
+**Note**:
+- For most hyervisor VT-x/AMD-v virtualization must be enabled in BIOS. For Hyper-V, however,
 it needs to be disabled.
+- We recommend to use `Virtualbox >= 5.1.12` on Windows to avoid [known issue](./docs/troubleshooting.md/#troubleshooting.md#error-getting-state-for-host-machine-does-not-exist)
```
Comment (hferentschik) [inline README.md]: The link is wrong. I will adjust it when I merge...

**Comment by hferentschik** [comment]
Merged. Updated troubleshooring link while doing so

File: README.md (hunk 3 above)
Comment (budhrg) [inline README.md]: Is it require to pin to this version?   I am able to run in Virtualbox 5.0.30 provided I am running through C drive.

**Pull request closed without merging**